src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="http://localhost:8080/"
+    method="GET"
+    data-qa="big"
+    class="search__form">
```

Comment:
```suggestion
    <form
      action="http://localhost:8080/"
      method="GET"
      data-qa="big"
      class="search__form"
    >
```
point 5 of [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)